.github/workflows/iroha2-dev-pr-ui.yml

```diff
@@ -2,7 +2,7 @@ name: I2::Dev::Tests::UI
 
 on:
   pull_request:
-    branches: [iroha2-dev]
+    branches: [iroha2-stable, iroha2-lts]
```

Comment:
I'd add `dev` at least temporarily. If it's the only failing workflow we can live without it.